actions/class.Creator.php

```
@@ -119,6 +121,12 @@ public function index()
             $guidedNavigation = $runtimeConfig['guidedNavigation'];
         }
         $this->setData('guidedNavigation', json_encode($guidedNavigation == true));
+        $this->setData(
+            'itemCommentsEnabled',
+            $this->getFeatureFlagChecker()->isEnabled(
+                FeatureFlagCheckerInterface::FEATURE_FLAG_RESOURCE_COMMENTS_ENABLED
```

PHPStan (2.2.8)

[error] Access to undefined constant oat\tao\model\featureFlag\FeatureFlagCheckerInterface::FEATURE_FLAG_RESOURCE_COMMENTS_ENABLED.


(classConstant.notFound)